src/jdk.compiler/share/classes/com/sun/tools/javac/processing/JavacProcessingEnvironment.java

```diff
@@ -714,7 +714,7 @@ static class ProcessorState {
                         add(importStringToPattern(allowModules, annotationPattern,
                                                   processor, log, lint));
                     if (lint && !patternAdded) {
-                        log.warning(Warnings.ProcDuplicateSupportedAnnotation(annotationPattern,
+                        log.warning(PROCESSING, null, Warnings.ProcDuplicateSupportedAnnotation(annotationPattern,
```

Comment:
Did you run all langtools tests? I believe that if you change these, some test output will be broken, as an extra `[processing]` will be emitted. I'm seeing this while playing in a related area.

Reply:
Example: test/langtools/tools/doclint/AnonClassTest.java

Reply:
That seems ok. So probably there's something else broken in my repository. Tests do seem green with this PR, so that seems to point at a local issue on my part.